usr/share/rear/conf/default.conf

```diff
@@ -803,23 +803,96 @@ CLONE_ALL_USERS_GROUPS="no"
 
-# SSH_ROOT_PASSWORD defines a password for remote access to the recovery system as 'root' via SSH whithout requiring a
-# public/private key pair. This password is valid only while the recovery system is running and will not allow access
-# to a restored target system.
-# Be aware, the password is saved in hashed MD5 format (do not forget the password after months:)
-# Generate a hashed password with the following command:
-#   echo "my_rescue_root_password" | openssl passwd -1 -stdin
-# and copy paste the output of openssl to variable SSH_ROOT_PASSWORD='...' (mind the single quotes!)
-# into config file /etc/rear/local.conf
+##
+# SSH_FILES
+# SSH_ROOT_PASSWORD
+# SSH_UNPROTECTED_PRIVATE_KEYS
+#
+# What SSH files should be copied into the ReaR rescue/recovery system and
+# how SSH access in the ReaR rescue/recovery system should be set up.
+# When SSH_FILES is set to a 'false' value no SSH file is copied into the recovery system
+# and nothing is done to set up SSH (in particular no sshd) in the recovery system.
+# In this case one could still use generic ReaR functionality like COPY_AS_IS and
+# REQUIRED_PROGS to manually copy some plain SSH files into the recovery system
+# but then there is still none of the special SSH setup in the recovery system.
+# When SSH_FILES is set to a 'true' value all the "usual SSH files" in
+# /etc/ssh* /etc/openssh* /etc/centrifydc/ssh* /root/.ssh /root/.shosts
+# (including private SSH keys) get copied into the recovery system
+# to make things "just work" in the recovery system which usually makes
+# the recovery system (in particular its ISO image and any recovery medium)
+# a confidential piece of data that needs to be protected against unwated access
+# cf. https://github.com/rear/rear/issues/1511 and https://github.com/rear/rear/issues/1512
+# When SSH_FILES is an array of files or directories like SSH_FILES=( /etc/ssh /root/.ssh )
+# those files and directories are copied into the recovery system via COPY_AS_IS
+# so that also COPY_AS_IS_EXCLUDE works to exclude some unwanted files from that list
+# like COPY_AS_IS_EXCLUDE=( "${COPY_AS_IS_EXCLUDE[@]}" '/etc/ssh/*key*' '/root/.ssh/*key*' )
+# where in particular excluding '/etc/ssh/*key*' results that SSH host key files
+# will be generated anew in the recovery system as described below.
+# When SSH_FILES is not set to a 'false' value additionally some SSH programs
+# are copied into the recovery system, in particular ssh, sshd, scp, sftp, ssh-agent,
+# and ssh-keygen plus a sftp-server program (e.g. /usr/lib/ssh/sftp-server).
+# The default setting SSH_FILES="avoid_sensitive_files" should be reasonably secure
+# because it avoids copying sensitive SSH files as follows:
+# From /etc/ssh only moduli, ssh_config, sshd_config, and ssh_known_hosts are copied and
+# from /root/.ssh only authorized_keys and known_hosts are copied into the recovery system.
+# An insufficiently encrypted /root/.ssh/authorized_keys file is a security problem so that
```

Comment:
An `authorized_keys` file is not a security problem. See https://github.com/rear/rear/pull/1530#issuecomment-336407592.

Reply:
Done.